integration_tests/log4rs_integration_tests.yml

```diff
@@ -102,68 +102,109 @@ appenders:
     encoder:
       pattern: "{d(%Y-%m-%d %H:%M:%S.%f)} [{t}] {l:5} {m}{n}"
 
-# Set the default logging level to "warn" and attach the "stdout" appender to the root
+# root (to base_layer)
 root:
-  level: warn
+  level: trace
   appenders:
-    - stdout
+    - base_layer
 
 loggers:
-  # Route log events sent to the "core" logger to the "base_layer" appender
+  # console output
+  base_node:
+    level: warn
+    appenders:
+      - stdout
+    additive: false
+  # core (mainly base node)
   c:
+    level: trace
+    appenders:
+      - core
+    additive: false
+  c::cs::lmdb_db::lmdb_db:
+    level: trace
+    appenders:
+      - core
+    additive: false
+  c::pow::lwma_diff:
     level: debug
     appenders:
-      - base_layer
+      - core
+    additive: false
+  c::bn::pow::sha3:
+    level: debug
+    appenders:
+      - core
+    additive: false
+  tari::base_node::grpc:
+    level: trace
+    appenders:
+      - core
     additive: false
-  # Route log events sent to the "wallet" logger to the "base_layer" appender
+  # base_layer (mainly wallet)
```

Comment:
Feels weird to have the wallet be Base Layer but not core? If you are going to the trouble to relabel these various appenders then perhaps we should rename base_layer to wallet?

Reply:
Sorry, my mistake, this file should not have been added to the commit as it is what I sometimes use for testing; will revert to the previous version.